treestats_paper.tex

```diff
@@ -912,6 +933,9 @@ \section*{Application to 1000 Genomes tree sequences}
 and only aims to provide consistent tree topologies.
 The two methods that infer node times differ substantially,
 with Relate showing a significantly tighter agreement than GEVA.
+\krt{Does this make sense?  If geva is playing with node times based on ts inferred via tsinfer,
+    then a difference b/w it and relate is due to the difference in estimating the times, and not the
+details of the underlying ts, maybe?}
```

Comment:
I'm not sure what you mean here?

Reply:
I think this is a GitHub problem.  The comment refers to the sentence below, which asks if Relate is doing a better job at inferring the ts than is Reva, but the latter is not inferring the ts, just the node times.

Reply:
By GEVA I mean "tsinfer plus GEVA"; we should probably clarify that.